lib/stack_master.rb

```diff
@@ -121,6 +125,19 @@ def debug(message)
     stderr.puts "[DEBUG] #{message}".colorize(:green)
   end
 
+  def quiet!
+    @quiet = true
+  end
+  @quiet = QUIET_DEFAULT
+
+  def quiet?
+    @quiet
+  end
+
+  def reset_flags
+    @quiet = QUIET_DEFAULT
```

Comment:
Adding the following to this method:
```
    @non_interactive = NON_INTERACTIVE_DEFAULT
    @debug = DEBUG_DEFAULT
```

Results in random feature test failures...

```
    Then the stack "myapp-vpc" should contain this notification ARN "test_arn_1" # features/step_definitions/stack_steps.rb:54
      Stack does not exist (Aws::CloudFormation::Errors::ValidationError)
      ./lib/stack_master/test_driver/cloud_formation.rb:127:in `describe_stacks'
      ./features/step_definitions/stack_steps.rb:55:in `/^the stack "([^"]*)" should contain this notification ARN "([^"]*)"$/'
      features/stack_defaults.feature:77:in `Then the stack "myapp-vpc" should contain this notification ARN "test_arn_1"'
```

Reply:
Yeah, because it's a module variable it's persistent between tests - rspec doesn't unload/reload modules between individual tests, or even between files (god, could you imagine how slow that would be?!)

Reply:
workaround: add a `before(:each)` block to your `spec_helper` that resets these to defaults

Reply:
From @stevehodgkiss 
> `non_interactive` is setup once when the test suite loads (on requiring `stack_master/testing`), so resetting it would undo that change.